rfc/README.md

```diff
@@ -0,0 +1,46 @@
+# The OpenTofu RFC process
+
+If a larger body of work is needed for an enhancement in the [main OpenTofu repository](https://github.com/opentofu/opentofu) or other repositories, the core team often asks for an RFC to be written. We do this in order to make sure everyone is on the same page how a feature should be implemented. This repository holds all past RFCs and anyone can contribute RFCs.
+
+## Filing an RFC
+
+Filing and getting an RFC merged is intentionally a longer process. We use pull requests to collaboratively work on a technical solution to a problem.
+
+Before filing an RFC, ideally open an enhancement issue in the [main repository](https://github.com/opentofu/opentofu) in order to discuss if the feature has enough community support to warrant working on an RFC. This step is optional, but highly recommended.
```

Comment:
I'm hard pressed to think of a scenario in which we don't want to have a Issue open before work starts on an RFC?